tests/integration/targets/omevv_firmware/files/ansible_doc.txt

```diff
@@ -0,0 +1,202 @@
+- '        This module allows you to create, modify, or delete an'
+- '        OpenManage Enterprise Integration for VMware Center (OMEVV)'
+- '        firmware repository profile.'
+- ''
+- 'ADDED IN: version 9.8.0 of dellemc.openmanage'
```

Comment:
This should be 9.10.0

Reply:
This document seems to be of omevv firmware repository profile, please update this.

Reply:
updated